test/dynamo/test_torchxla_integration.py

```diff
@@ -125,9 +140,61 @@ def test_wrapper(self):
 
     return test_wrapper
 
+def training_compiler(gm, example_inputs):
+    @make_boxed_compiler
+    @disable
+    def fw_compiler(graph, inputs, *args, **kwargs):
+        return integration.extract_compiled_graph(graph, inputs)
+
+    @make_boxed_compiler
+    @disable
+    def bw_compiler(graph, inputs, *args, **kwargs):
```

Comment:
hmm, is it just separate so that you can insert print statements for debug? or any reason not to combine?

Reply:
actually, why do you need a special compiler for the test case?

Reply:
I think so. I must have adding this separate bw_compiler to adding print/breakpoint. I can remove it for simplicity.

Call `integration.extract_compiled_graph` in the compiler function so we can execute thru the `trace_once` bridge in the test case.